bower.json

```diff
@@ -24,7 +24,8 @@
     "lodash": "~4.5.1",
     "highcharts": "^4.2.3",
     "highcharts-ng": "^0.0.11",
-    "pdfjs-dist": "^1.4.121"
+    "pdfjs-dist": "^1.4.121",
+    "angular-ui-router-default": "^0.0.3"
```

Comment:
pdfjs-dist isn't new but it is what we use to parse the pdfs on the client. It was added with the change that moved parsing from the server to the client.